docs/nc-convert.md

```diff
@@ -53,6 +55,11 @@ You can also use a JSON manifest file to convert many files at once:
 `nc-convert` will skip files that are already up-to-date when using a manifest.
 Relative paths within `globalOptions` are interpreted relative to the manifest.
 
+Source files enumerating assets in a manifest can be automatically generated with:
+```
+> nc-convert -g manifest.json -o out/dir -r out::namespace
```

Comment:
The `out::namespace` here is not really intuitive for me. I get what it's saying after seeing the comment on how to use in Prefabs, then trying it out. 

I wonder if we could add an example comment like:

`Example: nc-convert -g sample/assets/manifest.json -o sample/source/shared -r nc::sample`